src/middlewared/middlewared/plugins/container_runtime_interface/images.py

```diff
@@ -196,18 +162,17 @@ def normalise_tag(self, tag):
 
     @private
     def get_system_images_tags(self):
-        with open(DEFAULT_DOCKER_IMAGES_LIST_PATH, 'r') as f:
-            images = [i for i in map(str.strip, f.readlines()) if i]
+        # TODO: Let's add coredns/pause image etc
 
-        images.extend([
+        images = [
             'nvidia/k8s-device-plugin:1.0.0-beta6',
```

Comment:
This should be moved to a non-docker host, as this might hit docker rate-limits.

(OpenEBS is excluded from dockerhub rate limits, this one is not)